tests/primeqa/mrc/metrics/rouge/test_rouge.py

```diff
@@ -0,0 +1,36 @@
+import datasets
+import pytest
+
+from tests.primeqa.mrc.common.base import UnitTest
+from primeqa.mrc.metrics.rouge.rouge import ROUGE
+
+class TestROUGE(UnitTest):
+    
+    @pytest.fixture(scope='session')
+    def metric(self):
+        return ROUGE()
+    
+    @pytest.fixture(scope='session')
+    def references_and_predictions(self):
+        references = [
+            dict(id='1', answers=["hello there"]),
```

Comment:
add a test when not equal?

Reply:
We decided this test is sufficient for now.